templates/claim_payment.html

```diff
@@ -8,21 +8,24 @@ <h4>Expiry Date:</h4>
             <p>{{payment.proposal.date_expiry}}</p>
             <h4>Message:</h4>
             <p>{{payment.message}}</p>
-            <h4>Claim form:</h4>
+            <h4>Claim payment:</h4>
             <form method="post">
             {%if not recipient%}
-                <input type="text" name="recipient" placeholder="Wallet address"/>
+                <div class="card-body">
+                    <a href="{{ url }}">{{ qrcode_svg | safe }}</a><br/>
+                    <p>SCAN or CLICK the QRCode to claim your zap.</p><br/>
```

Comment:
change to `SCAN or CLICK the QR code to claim your {{config["ASSET_NAME"]}}.`

Reply:
changed to the one above.